### PR TITLE
Add env load command.

### PR DESCRIPTION
**Pull request by public**
For loading env-files so you don't need to worry about secrets being exposed via `ps` or shell history.

Literally the first Go code I've written. I guess this could use some tests.

I also get tons of changes if I run `godep save` to add the new dependency on `opts`. Am I doing it wrong?

### CODE REVIEW

**Comment by ejholmes** [comment]
This is pretty cool! I've been doing this in the past:

``` console
$ emp set $(cat .env) -a <app>
```

Or, to copy environment from another app:

``` console
$ emp set $(emp env -a <other app>) -a <app>
```

Is there a security advantage to providing a command like this?

/cc @phobologic

**Comment by ejholmes** [comment]
To godep save, you'll need to do a `godep restore` first, then `godep save` should work.

File: env.go [hunk 1]
```diff
@@ -130,3 +132,44 @@ func runUnset(cmd *Command, args []string) {
 	must(err)
 	log.Printf("Unset env vars and restarted %s.", appname)
 }
+
+var cmdLoad = &Command{
+	Run:      runLoad,
+	Usage:    "load <file>",
```
Comment (phobologic) [inline env.go]: Can we change this to env-load just to make it a little more specific & matching the rest of the CLI?

**Comment by phobologic** [comment]
There's a small, edge case benefit to doing this over the standard cat way - if someone were to manage to run `ps` while the command was running (split second) they would see all the variables.

That said, if they were on the box already they're only a little ways away from getting at the file itself most likely :)  Either way, I think this isn't a bad update - just think the command name should be more specific.

Thanks @public!

File: env.go (hunk 1 above)
Comment (public) [inline env.go]: Yeah I regretted this name as soon as I made the PR :)

**Comment by ejholmes** [comment]
So, since `$(cat .env)` will expand, it's possible to see the variables with `ps`.

What do you think about making the `env` command read from stdin if no args are passed? e.g.

``` console
$ cat .env | emp set -a <app>
```

**Comment by public** [comment]
What if you aren't invoking `emp` from a shell?

**Comment by ejholmes** [comment]
@public good point! I think this is good (although, probably after the name change that @phobologic) suggested. Thanks!

File: env.go [hunk 2]
```diff
@@ -130,3 +132,44 @@ func runUnset(cmd *Command, args []string) {
...
+	NeedsApp: true,
+	Category: "config",
+	Short:    "load env file",
+	Long: `
+Loads environment variables from a file.
+
+Example:
+
+    $ emp load app.env
+    Set env vars from app.env and restarted myapp.
```
Comment (ejholmes) [inline env.go:148]: Little typo here: `s/restarted/restart`

Comment (public) [inline env.go:148]: I don't think so? They all say "restarted". It's an example output.

**Comment by public** [comment]
Updated.

**Comment by public** [comment]
I still get tons of weird godep changes. It's changing everything from Godep paths to real ones. e.g.

```
-       "github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
+       "github.com/bgentry/heroku-go"
```

it also adds a bunch of extra modules to the Godeps.json. Maybe I'm running a different godeps version to you?

**Comment by ejholmes** [comment]
Ah, sorry about that. You'll need to godep save with the `-r` flag to rewrite the import paths (so people can just `go get` it without needing godep):

``` console
$ godep save -r ./...
```

**Comment by public** [comment]
That makes more sense thanks. How is that kind of thing usually communicated in Go projects?

**Comment by public** [comment]
Rebuilt with `godep save -r ./...`. Looks more plausible now :)

**Comment by public** [comment]
Bah not having much luck with this :( `godep restore` no longer seems to actually download the `opts` package from Docker.

**Comment by ejholmes** [comment]
Looks like you're just missing a `git add Godeps/_workspace/src` and then it should build in CI.

I added a [CONTRIBUTING.md](https://github.com/remind101/emp/blob/master/CONTRIBUTING.md) file with instructions for this.

**Comment by public** [comment]
`godep save -r ./...` isn't adding the opts package to the workspace.

**Comment by public** [comment]
Trashing my entire Godeps folder and saving it again seemed to help. Seems a bit of a large hammer though :-/

**Comment by ejholmes** [comment]
Looks good. Thanks for this @public!

**Comment by public** [comment]
Great. I'll get started on Git Hub Enterprise next :)

**Pull request merged**